analysis/plot_utils.py

```diff
@@ -0,0 +1,475 @@
+import numpy as np
+import pandas as pd
+import glob
+import matplotlib.pyplot as plt
+from functools import reduce
+
+from sklearn.metrics import roc_curve, auc
+from sklearn.preprocessing import label_binarize
+
+
+POS_MAP = [(8,4), #0
```

Comment:
Any way to generalize this or configurable to arbitrary detector geometry (e.g. upcoming short-tank, WCTE, Super-K, Hyper-K)?

Reply:
yes, I think these plot utils should be significantly rewritten really, they're just copied from the old framework for now

Reply:
but I think that doesn't need to be done for this PR since we wanted to focus on getting something usable for others before tidying up the parts of the code that aren't the actual ML engine code

Reply:
btw this specific code is for placing PMTs of a single mPMT onto pixels for plotting, it's not placing the mPMT modules of the detector themselves, so it will be the same for every mPMT detector geometry and not needed for geometries without mPMTs. But the code in general is still not generic to different detectors.